cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -2198,6 +2213,9 @@ def _create_course(test, course_key, course_data):
     """
     Creates a course via an AJAX request and verifies the URL returned in the response.
     """
+    if course_key.deprecated:
+        raise SkipTest('OldMongo Deprecation')
```

Comment:
Could you please add a comment explaining how/why this happens?

Reply:
Skips is added due to the return of HttpResponseNotFound in course_handler for old mongo courses. Comment added.
Skips will be removed in future PRs.